aspnetcore/tutorials/signalr-typescript-webpack/sample/3.x/snippets/index1.ts

```diff
@@ -8,7 +8,7 @@ const btnSend: HTMLButtonElement = document.querySelector("#btnSend");
 const username = new Date().getTime();
 
 tbMessage.addEventListener("keyup", (e: KeyboardEvent) => {
-    if (e.keyCode === 13) {
+    if (e.key === 'Enter') {
```

Comment:
```suggestion
    if (e.key === "Enter") {
```

Reply:
Thanks Scott.  Actually, I did already make those changes earlier in the app sample, however this particular .ts was a snippet copy this doc uses that I forgot also required the same change.  Thanks for catching this!